pavement.py

```diff
@@ -73,9 +73,22 @@ def make_venv():
     bootstrap_venv(__here__ / path('env'), 'sideboard')
     develop_sideboard()
 
+def install_pip_requirements_in_dir(dir_of_requirements_txt):
+    path_to_pip = __here__ / path('env/bin/pip3')
```

Comment:
ok copy that.  i can try and figure out if there's a pip command that runs the right version, 2 or 3, depending on how the venv was installed.  or maybe we can query the environment for that